src/install/install.zig

```diff
@@ -5324,6 +5324,11 @@ pub const PackageManager = struct {
         \\<STR> ...                         "name" uninstall package as a link
     );
 
+    pub const pm_params = install_params ++ clap.parseParamsComptime(
+        \\-a, --all                         List all dependencies, including aliased (bun pm ls only)
```

Comment:
can we make it only show for `bun pm ls --help`?

Reply:
In the current structure, not using clap's printer as far as I'm aware.  If we refactored to have commands and subcommands, all with their own arguments, then yes.